add_to_app/MultipleFluttersIos/MultipleFluttersIos/SingleFlutterViewController.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import Flutter
+
+class SingleFlutterViewController : FlutterViewController, DataModelObserver {
+  var _channel : FlutterMethodChannel?
+  var _engine : FlutterEngine!
+  
+  init(withEntrypoint entryPoint: String?) {
+    let appDelegate : AppDelegate = UIApplication.shared.delegate as! AppDelegate
+    _engine = appDelegate.engines.makeEngine(withEntrypoint: entryPoint, libraryURI: nil)
+    super.init(engine: _engine, nibName: nil, bundle: nil)
+    DataModel.shared.addObserver(observer: self);
+  }
+  
+  deinit {
+    DataModel.shared.removeObserver(observer: self)
+  }
+  
+  required init(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+  
+  func onCountUpdate(newCount: Int64) {
+    if let channel = _channel {
+      channel.invokeMethod("setCount", arguments: newCount);
```

Comment:
isn't this circular with the `count` setter below?

Reply:
I don't think so, I don't see it and it doesn't deadlock.  What cycle were you thinking?  This tells dart to update its count, there isn't a message that comes back from dart.

Reply:
Let's move thread. This is a subset of the comment in https://github.com/flutter/samples/pull/670#discussion_r563205815.